src/project/project.controller.ts

```diff
@@ -23,4 +28,24 @@ export class ProjectController {
   public async getAll(): Promise<ProjectDTO[]> {
     return await this.serv.getAll();
   }
+
+  @Get(':id')
+  public async getById(id: string): Promise<ProjectDTO> {
```

Comment:
```suggestion
  public async findById(id: string): Promise<ProjectDTO> {
```
In another controller the same name was with PascalCase and here is with camelCase